custom_components/lutron_caseta_pro/__init__.py

```diff
@@ -185,10 +186,22 @@ async def async_setup_bridge(hass, config, fname, bridge):
     for device in devices:
         types[device["type"]].append(device)
 
-    # load MAC address used for unique IDs
-    mac_address = None
+    # load the bridge's MAC address used for generating unique IDs for each device
+    host = bridge[CONF_HOST]
+    mac_address = getmac.get_mac_address(ip=host)
```

Comment:
This does I/O and can also throw an exception.  It should be wrapped in `await hass.async_add_executor_job`. ... probably will need `partial` as well due to the keyword arg.